CRM/Core/BAO/UFMatch.php

```diff
@@ -624,21 +624,20 @@ public static function getUFValues($ufID = NULL) {
       return [];
     }
 
-    static $ufValues;
-    if ($ufID && !isset($ufValues[$ufID])) {
+    if ($ufID && !isset(Civi::$statics[__CLASS__][__FUNCTION__][$ufID])) {
```

Comment:
This is fine, but FYI the `if ($ufID` part is redundant with the `if (!$ufID)` above.

Reply:
Right. I'll update.